bin/spec-md

```diff
@@ -1,41 +1,61 @@
 #!/usr/bin/env node
 
-var fs = require('fs');
 var path = require('path');
 var specmd = require('../');
+var yargs = require('yargs');
```

Comment:
Could you please remove yargs and use the existing simplified argument parsing? My attempt to keep dependencies low and simple